### PR TITLE
Node - Week4 Assignment

### PR DESCRIPTION
**Pull request by AnnaSolovykh**

### CODE REVIEW

File: 02-express-tutorial/app.js
Comment (leo-mutuku) [inline 02-express-tutorial/app.js:1]: Well refactored, 

#  The below express functions should be call before the routes

app.use(express.static('./methods-public'));
app.use(express.urlencoded( {extended: false} ));
app.use(express.json());
app.use(cookieParser());

File: 02-express-tutorial/routes/people.js
Comment (leo-mutuku) [inline 02-express-tutorial/routes/people.js:1]: people api routes well defined and implimented

**Review by leo-mutuku — APPROVED** [review]

File: 02-express-tutorial/controllers/people.js
Comment (leo-mutuku) [inline 02-express-tutorial/controllers/people.js:1]: People controllers CRUDE well implimented

**Review by leo-mutuku — APPROVED** [review]

File: 02-express-tutorial/methods-public/javascript.html
Comment (leo-mutuku) [inline 02-express-tutorial/methods-public/javascript.html:1]: Api version number updated

**Review by leo-mutuku — APPROVED** [review]

**Review by leo-mutuku — APPROVED** [review]

**Review by leo-mutuku — APPROVED** [review]

**Review by leo-mutuku — APPROVED** [review]

**Review by leo-mutuku — APPROVED** [review]

**Review by leo-mutuku — APPROVED** [review]
Request params, get person by id well implimented

**Review by leo-mutuku — APPROVED** [review]

File: 02-express-tutorial/middleware/auth.js
Comment (leo-mutuku) [inline 02-express-tutorial/middleware/auth.js:1]: Auth middleware welll defined

File: 02-express-tutorial/controllers/auth.js
Comment (leo-mutuku) [inline 02-express-tutorial/controllers/auth.js:1]: Cookies well defined, and removed on log off

**Review by leo-mutuku — APPROVED** [review]

**Comment by leo-mutuku** [comment]
Didn't see product api, approved as it is the same logic as people api. Confirm if you skip that part.

**Comment by AnnaSolovykh** [comment]
> Didn't see product api, approved as it is the same logic as people api. Confirm if you skip that part.

This is from the previous home assignment, so I deleted it in the very first commit. Should I keep? 
Another thing popped up into my mind - should I keep and refactor for practice?..

**Comment by AnnaSolovykh** [comment]
Done! :)

File: 02-express-tutorial/app.js
Comment (leo-mutuku) [inline 02-express-tutorial/app.js:1]: Great for the quick update, express middleware well defined and declared

Comment (leo-mutuku) [inline 02-express-tutorial/app.js:1]: Products route well defined

File: 02-express-tutorial/controllers/products.js
Comment (leo-mutuku) [inline 02-express-tutorial/controllers/products.js:1]: Product controller get operations well implimented

File: 02-express-tutorial/routes/products.js
Comment (leo-mutuku) [inline 02-express-tutorial/routes/products.js:1]: Products route well defined and exported to the express middleware

**Review by leo-mutuku — APPROVED** [review]

**Comment by leo-mutuku** [comment]
Well done! Thanks for the quicks handons.

**Comment by leo-mutuku** [comment]
You can merge the branch.

**Comment by AnnaSolovykh** [comment]
> Well done! Thanks for the quicks handons. 

Thank you for the quick reviews! I really appreciate it!

**Pull request merged**